http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HeadersIT.java

```diff
@@ -93,6 +96,19 @@ void testPathSpecificHeaderRulesOrder() {
         cacheControlMatches(response, "max-age=1");
     }
 
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/pull/41411")
+    void testWithNoAcceptHeader() {
```

Comment:
As this is non reactive it should be probably disabled. If this work you should probably comment on that issue some findings